docs/howto/repeatuntil.adoc

```diff
@@ -7,8 +7,9 @@ The `repeat-until` command will re-run the commands in `then` until the named `s
   - sleep: 1m #add a delay or the command run very quickly
   - ...
 ```
-The `signal` can come from a different script or from a child of the `repeat-until`
+The `signal` can come from a different script or from a child of the `repeat-until`. If the `signal` comes from within the script then we normally `set-signal: ready 1` beforehand to ensure the signal is initialized before looping
 ```yaml
+- set-signal: ready 1
```

Comment:
What does "ready 1" mean ?

Reply:
ready is the name of the new signal and 1 is the expected number of calls to signal. It isn't strictly necessary because the call to `signal` inside the `repeat-until`